### DISCUSSION TITLE
Reportee field in GitHub Projects

### DISCUSSION THREAD

**Discussion by sabareesh-r23** [community/community#78178] [Projects and Issues]
A user creates an issue, and the entire bug lifecycle is monitored within the GitHub project. Once the issue is resolved, its status transitions from Dev In Progress to Dev Complete (Custom field in GitHub). Now, is there a method for a user to easily identify and view the modified issue, especially in scenarios with numerous issues within a specific project? Additionally, if I wish to filter and display only the issues logged or reported/authored by me among the available project views, how can I achieve this at scale?

There is an option to filter by assignee:@me but that's basically for the developer who would be working on that . How will I be viewing the dashboard without using an assignee?

**Comment by erickdc7** [comment]
To easily identify and view modified issues within a GitHub project, especially when there are numerous issues, you can use GitHub's filtering capabilities. [While GitHub Projects allows you to filter items by various metadata such as assignees and labels, it does not currently support filtering issues by the reporter or author directly](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects).

However, you can achieve a similar result by using the search functionality within the Issues or Pull Requests section of the repository. You can filter issues that you've created by using the `author` qualifier in the search bar. For example, you can type `author:@me` to display only the issues that you have reported.

If you want to view these filtered issues within the GitHub Projects dashboard, you can create a custom view that includes the issues you're interested in. While creating or editing a view, you can specify the filters you want to apply. Although you cannot directly filter by the reporter, you can use other criteria such as labels or milestones that you may have associated with the issues you reported.

For more detailed information on filtering issues and pull requests, you can refer to the GitHub Docs. Additionally, for filtering projects and managing views, the [GitHub Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) provide guidance on customizing views using filters for item metadata and fields in your project.

Remember, the filtering capabilities are subject to the features provided by GitHub, and certain specific filters like filtering by reporter may not be available. If you need this feature at scale and it's not currently supported, consider reaching out to GitHub support or suggesting it as a feature request.
